src/Tribe/Integrations/ACF/ACF.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Class Tribe__Events__Integrations__ACF__ACF
+ *
+ * Handles the integration between The Events Calendar plugin and Advanced Custom Fields.
+ *
+ * This class is meant to be an entry point hooking specialized classes and not
+ * a logic hub per se.
+ */
+class Tribe__Events__Integrations__ACF__ACF {
+
+	/**
+	 * @var Tribe__Events__Integrations__ACF__ACF
+	 */
+	protected static $instance;
+
+	/**
+	 * @return Tribe__Events__Integrations__ACF__ACF
+	 */
+	public static function instance() {
+		if ( empty( self::$instance ) ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Hooks the filters and actions neede for this integration to work.
+	 *
+	 * @since TBD
+	 */
+	public function hook() {
+		add_action( 'admin_enqueue_scripts', array( $this, 'load_compat_js' ) );
+	}
+
+	/**
+	 * Load our compatability JS script to supplement the events-admin.js script.
+	 *
+	 * @since TBD
+	 */
+	public function load_compat_js() {
```

Comment:
Instead of having a method like this and adding it via an action, we could use `tribe_asset()`. It will handle all the funkiness for you, like working when this is a must-use plugin and it will load the minified file when appropriate!

Reply:
i think tribe_asset only loads from the `resources/` dir, while my compat script is in the `integrations` dir — i also don't have a minified script

Reply:
I'm pretty certain it can load from anywhere within the content dir, and a few other rarely configured dirs. It should be flexible enough for this.

Reply:
See: https://github.com/moderntribe/tribe-common/blob/ec134e9ff26597246f1ca0de880c4dd927b9563d/src/Tribe/Assets.php#L216